arangosh/Shell/ClientFeature.cpp

```diff
@@ -192,15 +205,21 @@ void ClientFeature::validateOptions(std::shared_ptr<ProgramOptions> options) {
 
   _haveServerPassword = !options->processingResult().touched("server.password");
 
-  if (_askJwtSecret && options->processingResult().touched("server.password")) {
+  if ((_askJwtSecret | hasJwtSecretFile) && options->processingResult().touched("server.password")) {
```

Comment:
can we use logical or instead?
```suggestion
  if ((_askJwtSecret || hasJwtSecretFile) && options->processingResult().touched("server.password")) {
```